File: luiz020/programacao_basica/questao05.py
print("** Seguradora XYZ **")
print("    >> Sistema de avalaiação <<")

contrato_sim = int(input("Qtd de contratos simples: "))

contrato_com = int(input("Qtd de contratos completos: "))

lucro = contrato_com * 350 + contrato_sim * 230

print("Lucro obtido: R$ " + str(lucro))

File: luiz020/programacao_basica/Questao05.java
import java.util.Scanner;

public class Questao05 {
    public static void main(String[] args) {
        int simples;
        int completo;
        int lucro;
        Scanner k;
        k = new Scanner (System.in);

        System.out.println("** Seguradora XYZ **");
        System.out.println("    >> Sistema de avaliação <<");

        System.out.print("Quantidade de contratos simples: ");
        simples = k.nextInt();

        System.out.print("Quantidade de contratos completos: ");
        completo = k.nextInt();

        lucro = simples * 2300 + completo * 3500;

        System.out.println("Lucro obtido: R$ " + (lucro));

        }}
